### PR TITLE
German translation for dekstop file

### PR DESCRIPTION
**Pull request by dadosch**

### CODE REVIEW

File: gqrx.desktop [hunk 1]
```diff
@@ -8,6 +8,7 @@ GenericName[ru]=Программно-определённое радио
 Comment[ru]=Приемник для программно-определенного радио (SDR) использующий GNU Radio и библиотеку Qt.
 GenericName[nl]=Software Defined Radio
 Comment[nl]=Software defined radio ontvanger geïmplementeerd met GNU Radio en de Qt GUI toolkit
+Commenr[de]=Software defined Radio implementiert durch GNU Radio und das Qt GUI Toolkit
```
Comment (alexf91) [inline gqrx.desktop]: I'd suggest:
Comment[de]=Software Defined Radio auf Basis von GNU Radio und dem Qt GUI Toolkit

Sounds more natural to me.

Comment (dadosch) [inline gqrx.desktop]: Yeah, your' right.

**Comment by csete** [comment]
Thanks!

**Comment by alexf91** [comment]
There's still a typo in "Comment".

**Comment by csete** [comment]
Oh, I missed that :-)
It should be corrected now.

**Pull request merged**